google-cloud-spanner/lib/google/cloud/spanner/convert.rb

```diff
@@ -183,7 +183,9 @@ def grpc_type_for_field field
             when :PG_NUMERIC
               V1::Type.new(code: :NUMERIC, type_annotation: :PG_NUMERIC)
             when :PG_JSONB
-              V1::Type.new(code: :JSON, type_annotation: :PG_JSONB)  
+              V1::Type.new(code: :JSON, type_annotation: :PG_JSONB)
+            when :PG_OID
```

Comment:
Can we also add a unit test in 
`ruby-spanner/google-cloud-spanner/test/google/cloud/spanner/convert/grpc_type_for_field_test.rb`